ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,458 +1,470 @@
       route-target import {{ route_target }}
-{%                endfor %}
-{%            endif %}
-{%            if router_bgp.vlan_aware_bundles[vlan_aware_bundle].route_targets.export is defined and router_bgp.vlan_aware_bundles[vlan_aware_bundle].route_targets.export is not none %}
-{%                for route_target in router_bgp.vlan_aware_bundles[vlan_aware_bundle].route_targets.export %}
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.vlan_aware_bundles[vlan_aware_bundle].route_targets.export is arista.avd.defined %}
+{%             for route_target in router_bgp.vlan_aware_bundles[vlan_aware_bundle].route_targets.export %}
       route-target export {{ route_target }}
-{%                endfor %}
-{%            endif %}
-{%            for redistribute_route in router_bgp.vlan_aware_bundles[vlan_aware_bundle].redistribute_routes | arista.avd.natural_sort %}
+{%             endfor %}
+{%         endif %}
+{%         for redistribute_route in router_bgp.vlan_aware_bundles[vlan_aware_bundle].redistribute_routes | arista.avd.natural_sort %}
       redistribute {{ redistribute_route }}
-{%            endfor %}
-      vlan {{ router_bgp.vlan_aware_bundles[vlan_aware_bundle].vlan }}
 {%         endfor %}
-{%     endif %}
+      vlan {{ router_bgp.vlan_aware_bundles[vlan_aware_bundle].vlan }}
+{%     endfor %}
 {# address families activation #}
 {# address family evpn activation #}
-{%     if router_bgp.address_family_evpn is defined and router_bgp.address_family_evpn is not none %}
+{%     if router_bgp.address_family_evpn is arista.avd.defined %}
    !
    address-family evpn
-{%       if router_bgp.address_family_evpn.peer_groups is defined and router_bgp.address_family_evpn.peer_groups is not none %}
-{%          for peer_group in router_bgp.address_family_evpn.peer_groups | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in is defined and router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in is not none %}
+{%         for peer_group in router_bgp.address_family_evpn.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out is defined and router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out is not none %}
+{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out }} out
 {%             endif %}
 {%             if router_bgp.address_family_evpn.peer_groups[peer_group].activate == true %}
```

Comment:
No check if `router_bgp.address_family_evpn.peer_groups[peer_group].activate` is defined